.github/workflows/pr.yml

```diff
@@ -13,6 +13,7 @@ jobs:
     - uses: actions-rs/toolchain@v1
       with:
         components: rustfmt, clippy
+    - uses: Swatinem/rust-cache@v1
```

Comment:
Good catch! I'm updating the PR to pin to the specific revision of the action, to avoid the first problem. As for the second -- note that this is only set up for the lint action which runs clippy / fmt, which I _assume_ uses debug builds. I wasn't sure how to make the cache work for building in Docker, so it's not enabled for the test and release builds yet.